webapp/src/components/jira_ticket_tooltip/jira_ticket_tooltip.tsx

```diff
@@ -0,0 +1,279 @@
+                            />
+                        </a>
+                    </div>
+                </div>
+                <div className={'ticket-popover-body'}>
+                    <div className={'ticket-popover-title'}>
+                        <a href={jiraTicketURI}>
+                            <h5>{this.truncateString(jiraTicketTitle, 80)}</h5>
+                        </a>
+                        {this.tagTicketStatus(jiraTicketStatusName)}
+                    </div>
+                    <div className={'ticket-popover-description'}>
+                        {jiraTicketDescription}
+                    </div>
+                    <div className={'ticket-popover-labels'}>
+                        {this.fixVersionLabel(jiraTicketVersions)}
+                        {this.labelList(jiraTicketLabels)}
+                    </div>
+                </div>
+                <div className={'ticket-popover-footer'}>
+                    {jiraTicketAssigneeAvatarURI ?
+                        (
+                            <img
+                                className={'ticket-popover-footer-assigner-profile'}
+                                src={jiraTicketAssigneeAvatarURI}
+                                alt={'jira assigner profile'}
+                            />
+                        ) :
+                        (
+                            <span className='default-avatar'>
+                                <svg
+                                    width='18'
+                                    height='18'
+                                    viewBox='0 0 18 18'
+                                    role='presentation'
+                                >
+                                    <g
+                                        fill='white'
+                                        fillRule='evenodd'
+                                    >
+                                        <path
+                                            d='M3.5 14c0-1.105.902-2 2.009-2h7.982c1.11 0 2.009.894 2.009 2.006v4.44c0 3.405-12 3.405-12 0V14z'
+                                        />
+                                        <circle
+                                            cx='9'
+                                            cy='6'
+                                            r='3.5'
+                                        />
+                                    </g>
+                                </svg>
+                            </span>
+                        )
+                    }
+                    {jiraTicketAssigneeName ?
+                        (
+                            <span>
+                                <span className={'ticket-popover-footer-assigner-name'}>
+                                    {jiraTicketAssigneeName}
+                                </span>
+                                <span className={'ticket-popover-footer-assigner-is-assigned'}>
```

Comment:
These classnames are wrong. Use the BEM convention. https://getbem.com/naming/
You can also take help from @sooraj-shukla @avas27JTG .